src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -0,0 +1,206 @@
+        public CompositeChangeToken(System.Collections.Generic.IReadOnlyList<Microsoft.Extensions.Primitives.IChangeToken> changeTokens) { }
+        public bool ActiveChangeCallbacks { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Microsoft.Extensions.Primitives.IChangeToken> ChangeTokens { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public bool HasChanged { get { throw null; } }
+        public System.IDisposable RegisterChangeCallback(System.Action<object> callback, object state) { throw null; }
+    }
+    public static partial class Extensions
+    {
+        public static System.Text.StringBuilder Append(this System.Text.StringBuilder builder, Microsoft.Extensions.Primitives.StringSegment segment) { throw null; }
+    }
+    public partial interface IChangeToken
+    {
+        bool ActiveChangeCallbacks { get; }
+        bool HasChanged { get; }
+        System.IDisposable RegisterChangeCallback(System.Action<object> callback, object state);
+    }
+    [System.Diagnostics.DebuggerDisplayAttribute("Value = {_value}")]
+    [System.ObsoleteAttribute("This type is obsolete and will be removed in a future version.")]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct InplaceStringBuilder
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public InplaceStringBuilder(int capacity) { throw null; }
+        public int Capacity { get { throw null; } set { } }
+        public void Append(Microsoft.Extensions.Primitives.StringSegment segment) { }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public void Append(char c) { }
+        public void Append(string value) { }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public void Append(string value, int offset, int count) { }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct StringSegment : System.IEquatable<Microsoft.Extensions.Primitives.StringSegment>, System.IEquatable<string>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public static readonly Microsoft.Extensions.Primitives.StringSegment Empty;
+        public StringSegment(string buffer) { throw null; }
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]public StringSegment(string buffer, int offset, int length) { throw null; }
+        public string Buffer { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
```

Comment:
Do we need the `CompilerGeneratedAttribute`? I don't think so. @ericstj ?

Reply:
I think this source should be regenerated with dotnet/runtime settings.  This source came from aspnet/extensions